pandas/tests/groupby/test_whitelist.py

```diff
@@ -164,33 +164,45 @@ def raw_frame():
 @pytest.mark.parametrize('axis', [0, 1])
 @pytest.mark.parametrize('skipna', [True, False])
 @pytest.mark.parametrize('sort', [True, False])
+@pytest.mark.parametrize('as_index', [True, False])
 def test_regression_whitelist_methods(
         raw_frame, op, level,
-        axis, skipna, sort):
+        axis, skipna, sort, as_index):
     # GH6944
     # GH 17537
     # explicitly test the whitelist methods
 
+    if not as_index and axis == 1:
+        pytest.skip('as_index=False only valid for axis=0')
+
     if axis == 0:
         frame = raw_frame
     else:
         frame = raw_frame.T
 
+    groupby_kwargs = {'level': level, 'axis': axis, 'sort': sort,
```

Comment:
I don't think you actually need to define this kwargs, just inline it when calling frame.groupby(...) no?

Reply:
Ah, good point